router/src/main/java/io/micronaut/web/router/version/Version.java

```diff
@@ -0,0 +1,21 @@
+package io.micronaut.web.router.version;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to specify the version of route.
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Version {
```

Comment:
I think this is a common enough annotation name it could go into the `core` module. I would put it in `io.micronaut.core.version.annotation`